apicast/src/configuration/service.lua

```diff
@@ -137,7 +137,7 @@ function backend_version_credentials.version_oauth(config)
   -- Resource servers MUST support this method. [Bearer]
   access_token = access_token or authorization.token
 
-  local o = oauth.new(self)
+  local o = oauth.new(_M)
```

Comment:
because `self` does not exist in this function ?

also looks like passing whatever (_M) has no effect on resulting method call, so maybe that method call should not be instance method requiring those parameters.